packages/ui/src/ReservoirKitProvider.tsx

```diff
@@ -16,6 +16,7 @@ import { SWRConfig } from 'swr'
 export type CoinGecko = {
   proxy?: string
   apiKey?: string
+  coinId?: string
```

Comment:
Hm, I think this would be better if we had an id to contract mapping, in case you encountered multiple coingecko currencies. Also this would override everything, including the mainnet ethereum currency, weth etc. We need to make it more specific.

Reply:
This should work below.
```ts
export type CoinId = {
   [key: string]: string
 }
 export type CoinGecko = {
   proxy?: string
   apiKey?: string
   coinIds?: CoinId
 }
 
 const config: CoinGecko = {
     coinIds: {
        eth:"123",
        weth:"1234"
     }
 }
 
 ```